locale/area_containers.eo.tr

```diff
@@ -0,0 +1,13 @@
+# textdomain: area_containers
+Area Container=Areujo
+Area Container Key=Areuja Ŝlosilo
+Area Container Lock=Areuja Ŝloso
+Blank Area Container Key=Malplena Areuja Ŝlosilo
+Broken Area Container=Rompita Areuja Ŝlosilo
```

Comment:
```suggestion
Broken Area Container=Rompita Areujo
```

I think this is right, since the phrase being translated is not "Broken Are Container Key."

Reply:
@jolesh Is this correct?

Reply:
Yeah, thanks. Looks like I missed that.
Thanks for proof reading :)